site/content/docs/main/release-instructions.md

```diff
@@ -1,16 +1,100 @@
 ---
 title: "Release Instructions"
 layout: docs
+toc: "true"
 ---
-
 This page covers the steps to perform when releasing a new version of Velero.
 
-## Preparing for a release
-
-### (GA Only) Create release blog post
-
-For each major or minor release, create and publish a blog post to let folks know what's new.
-
+## General notes
+- Please read the documented variables in each script to understand what they are for and how to properly format their values.
+- You will need to have a remote named `upstream` pointing to the Velero repository so the scripts can work properly.
+- GA release: major and minor releases only. Example: 1.0 (major), 1.5 (minor).
```

Comment:
There's also patch releases, for example v1.4.2 would be a GA patch release.

Reply:
I should go into a bit more detail here, actually.

Patch releases are usually the 2nd+ within a minor release series. Often it's v1.x.1 and later, but in the case of the v1.5 series, it will be v1.5.2.

These differ because they will have a `release-v1.x` branch to be cut from, because the assumption is `main` will have included features and refactorings that we don't want to backport wholesale.